content/07.pathogenesis.md

```diff
@@ -446,17 +445,29 @@ Data and empirically determined biological mechanisms inform models, while model
 Many current efforts to model R~t~ have also led to tools that assist the visualization of estimates in real time or over recent intervals [@url:https://epiforecasts.io/covid; @url:https://rt.live].
 These are valuable resources, yet it is also important to note that the estimates arise from models containing many assumptions and are dependent on the quality of the data they use, which varies widely by region.
 
-### Molecular Signatures and Transmission
+### Molecular Signatures, Transmission, and Variants of Concern
 
 Genetic variation in SARS-CoV-2 has been used to elucidate patterns over time and space.
 Mutations observed in individual SARS-CoV-2 genome sequences can be used to trace transmission patterns and have provided insights during outbreak investigations [@doi:10.1186/s12967-020-02344-6; @doi:10.1093/nsr/nwaa036; @doi:10.1016/j.meegid.2020.104351].
 Similar mutations observed in several patients may indicate that the patients belong to the same transmission group.
 The tracking of SARS-CoV-2 mutations is recognized as an essential tool for controlling future outbreaks and tracing the path of the spread of SARS-CoV-2.
-Efforts vary widely by country: the UK has coordinated has coordinated a national database of viral genomes [@doi:10/d5mg]; no such coordination has been achieved in the USA.
+Efforts vary widely by country: the UK has coordinated a national database of viral genomes [@doi:10/d5mg]; no such coordination has been achieved in the USA.
 Several studies used phylogenetic analysis to determine the source of local COVID-19 outbreaks in Connecticut (USA), [@doi:10.1016/j.cell.2020.04.021], the New York City area (USA) [@doi:10.1126/science.abc1917], and Iceland [@doi:10.1056/NEJMoa2006100].
 There is an ongoing effort to collect SARS-CoV-2 genomes throughout the COVID-19 outbreak, and as of January 18, 2021 more than 381,000 genome sequences have been collected from patients.<!-- TODO: consider auto-updating this -->
 The sequencing data can be found at GISAID [@https://www.gisaid.org/], NCBI [@https://www.ncbi.nlm.nih.gov/sars-cov-2/], and COVID-19 data portal [@https://www.covid19dataportal.org/].
 
+While the evolution of SARS-CoV-2 variants are discussed elsewhere within this special issue [], some of these variants may be associated with increased transmission of SARS-CoV-2. 
```

Comment:
Reference to evolution manuscript required.

Reply:
@agitter this seems great to me. I looked through that one today and I'm optimistic we can get it on arXiv soon-ish, although hard to tell exactly how things will line up with resubmission here.